backend/src/test/java/com/staccato/moment/domain/MomentImagesTest.java

```diff
@@ -50,44 +50,23 @@ void failUpdateMomentImages() {
                 .hasMessage("사진은 5장을 초과할 수 없습니다.");
     }
 
-    @DisplayName("사진들을 추가할 때 기존 사진이 포함되지 않은 경우 삭제 후 추가한다.")
+    @DisplayName("사진들의 순서를 변경할 때 기존 사진이 포함되지 않은 경우 삭제 후 추가한다.")
```

Comment:
기존 사진이 포함되어 있는지의 여부와 상관없이 전부 삭제 후 전부 추가하기 때문에, 이렇게 바꿔도 괜찮을 것 같아요!

```suggestion
    @DisplayName("사진들의 순서를 변경할 때 기존 사진을 전부 삭제 후 추가한다.")
```